laze-project.yml

```
@@ -557,6 +559,43 @@ modules:
         FEATURES:
           - riot-rs/wifi-esp
 
+  - name: executor-thread
+    help: use embassy executor within riot-rs-threads thread
+    provides_unique:
+      - executor
+    env:
+      global:
+        FEATURES:
+          - riot-rs/executor-thread
+
+  - name: executor-single-thread
+    help: use embassy executor within single "thread mode" thread
```

```suggestion
    help: use embassy executor within single "thread mode" thread
    context:
      - esp
```
Doesn't compile for nrf or rp.